index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Salim Abdulai| Portfolio | Software Developer</title>
+    <link rel="stylesheet" href="./static/css/style.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-pc">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-mobile">
+            <ul>
+                <li class="welcome-text"><p>Welcome</p></li>
+                <li class="menu-icon"><img src="./static/images/Icons/menu-mobile.png" alt="menu-bar"></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="primary-text">
+                <h1>Hey there. I’m Salim.<br><span class="highlight">I’m a software developer</span></h1>
+            </div>
+            <div class="supporting-text">
+                <p> I can help you build a product , feature or website Look through some of my work and <br> 
+                    experience! If you like what you see and have a project you need coded, <br>  
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="social-media">
+                <a href="#"><img src="./static/images/Icons/github.png" alt="github-image"></a>
+                <a href="#"><img src="./static/images/Icons/linkedIn.png" alt="linkedIn-image"></a>
+                <a href="#"><img src="./static/images/Icons/angel_list.png" alt="angel_list-image"></a>
+                <a href="#"><img src="./static/images/Icons/twitter.png" alt="twitter-image"></a>
+                <a href="#"><img src="./static/images/Icons/medium.png" alt="medium-image"></a>
+            </div>
```

Comment:
The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)